rules/set-fsgroup-value/raw.rego

```diff
@@ -13,7 +13,7 @@ deny[msga] {
     # check securityContext has fsGroup set properly
     not fsGroupSetProperly(pod.spec.securityContext)
 
-    path := "spec.securityContext.fsGroup"
+    path := "spec.securityContext"
```

Comment:
We can separate to 2 different cases - if object has `spec.securityContext.fsGroup` with an incorrect value, and if it doesn't exist